### DISCUSSION TITLE
Guest has not initialized the display (yet) GNS3/EVE-NG

### DISCUSSION THREAD

**Discussion by network-beg** [utmapp/UTM#5856] [macOS]
Hello everyone, 

I'm new to emulation/virtualization on macOS.

I'd like to explain my problem here, in the hope that you'll be able to solve it.

I'm using UTM, more precisely I'm emulating eve-ng on UTM to virtualize networks. 
The problem is that when I want to use VNC to monitor Windows 10, the window of vnc says "guest has not initialized the display (yet)".
I've looked everywhere for a solution but haven't found anything convincing.

config :
M2 pro. 
macOS Sonoma.
UTM 4.4.4

Is there a problem with virtualization? Knowing that kvm is not taken into account, could this be a reason for the problem?

**Comment by osy** [comment]
For some reason, your VNC is configured to use the non-primary display. This isn't a UTM issue but you should figure out why your VNC software is selecting the wrong display.

**Comment by FarbodM2010** [comment]
Same here with ubuntu. Been looking for a solution for days. Tell me if you found a way.

**Comment by mgh24** [comment]
Ditto here. 

The VMs I have installed work fine, but attempting to install a new VM, get the same message. 

Edit: I can open a distro from the UTM gallery, using the option to "open in UTM", and it installs perfectly. Just unable to install from a downloaded ARM ISO.

**Comment by sadans1** [comment]
I faced this issue in GNS3 and for me this resolved after changing enable_hardware_acceleration and require_hardware_acceleration from **true** to **false** in /etc/gns3/gns3_server.conf file.  I might think there will be some same settings in eve-ng too.
